mkdocs.yml

```diff
@@ -116,16 +115,21 @@ nav:
 - Events: 'events.md'
 - Support: 'support.md'
 - Testimonials: 'testimonials.md'
-- Legacy SAFE Template (v1):
-  - Overview: 'legacy-template-overview.md'
-  - Run and Build the app: 'legacy-template-fake.md'
-  - Deploy to Docker: 'legacy-template-docker.md'
-  - Deploy to App Service: 'legacy-template-appservice.md'
-  - Deploy to Google Cloud AppEngine: 'legacy-template-gcp-appengine.md'
-  - Deploy to Google Cloud Kubernetes Engine: 'legacy-template-gcp-kubernetes.md'
-  - Deploy to Heroku: 'legacy-template-heroku.md'
-  - Deploy to IIS: 'legacy-template-iis.md'
-  - Application Insights Integration: 'legacy-template-azure-ai.md'
-  - Registering with Azure: 'legacy-template-azure-registration.md'
-  - Registering with Google Cloud: 'legacy-template-google-cloud.md'
-  - Full Stack Debugging: 'legacy-feature-debugging.md'
+- Legacy SAFE Template (v2):
```

Comment:
Maybe not for this PR, but there's a nice plugin for mkdocs https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin which allows you to skip the Table of Contents in yaml and instead rely solely on contents of a given directory - I've been successfully using it in a commercial project, the only drawback is you need to be aware how sections get ordered within same directory